### PR TITLE
Cooling/RTD Device Updates

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- Add a new device for FDQ Flow Meter `FDQ` in `analog_signals.py`
- Make a new class for PPMs with FDQ Flow Meters: `PPMCOOL` currently serving IM3/4/5/6K4.
- Fix `KBOMirrorHE`; only needs one flow meter readout. These serve MR2/3K4
- Update `FFMirrorZ` to include chin guard RTDs serving MR4/5K4
- Add Class `KBOMirrorChin` for Optics currently without cooling but with Chin Guard RTDs. Serving MR3K2

## Motivation and Context

Newly installed flow meters and RTDs.

## How Has This Been Tested?

- `FDQ` class was ran locally in a hutch-python session with functions tested. The PLC still needs to stop writing the offset value, but that will be fixed in a different PLC PR for imagers in TMO.
- `PPMCOOL` was launched successfully in a typhos screen.
- `KBOMirrorHE` was launched successfully with MR2/3K4 - an issue of sharing a pressure readout here exist, currently 1 PV is being read out on MR2, but it needs to be readout on MR3 as well; for now, MR3 pressure readout PV shows disconnected because it doesnt exist.
- `KBOMirrorChin` was launched successfully in a typhos screen with MR3K2.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [ ] Test suite passes locally
- [ ] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/mirror.py [hunk 1]
```diff
@@ -1206,6 +1206,14 @@ class FFMirrorZ(FFMirror):
     # RMS Cpts:
     z_enc_rms = Cpt(PytmcSignal, ':ENC:Z:RMS', io='i', kind='normal')
 
+    # RTDs
+    chin_left_rtd = Cpt(PytmcSignal, ':RTD:CHIN:L', io='i',
+                        kind='normal')
+    chin_right_rtd = Cpt(PytmcSignal, ':RTD:CHIN:L', io='i',
```
Comment (klauer) [inline pcdsdevices/mirror.py]: (I know this is draft but I saw the diff and noticed the copy-paste error here: `right_rtd ... ':RTD:CHIN:L'`)

File: docs/source/upcoming_release_notes/1156-RTD-Cooling.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+--------
+- `KBOMirrorHE` in `mirror.py` only has 1 flow sensor per mirror, so remove one.
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1156-RTD-Cooling.rst:32]:
```suggestion
- nrwslac
```

File: pcdsdevices/analog_signals.py [hunk 3]
```diff
@@ -220,3 +221,28 @@ def tweak_mesh_voltage(self, delta_hv_sp, test_flag=False):
                 self.set_rel_mesh_voltage(-delta_hv_sp, wait=False)
             if test_flag:
                 return
+
+
+class FDQ(BaseInterface, Device):
+    """
+        A collection of PVs to inteface python with a Keyance FDQ Flow Meter.
```
Comment (tangkong) [inline pcdsdevices/analog_signals.py:228]:
```suggestion
    A collection of PVs to inteface python with a Keyance FDQ Flow Meter.
```

File: pcdsdevices/mirror.py [hunk 4]
```diff
@@ -1480,3 +1487,25 @@ class for storing pitch positions based on state
...
+
+class KBOMirrorChin(KBOMirror):
+    """
+    Kirkpatrick-Baez Mirror with Bender Axes.
+
+    1st gen Toyama designs with LCLS-II Beckhoff motion architecture.
+
+    With 2 RTDs installed on the chin guard.
+
+  Parameters
```
Comment (tangkong) [inline pcdsdevices/mirror.py:1500]:
```suggestion
    Parameters
```

**Review by tangkong — APPROVED** [review]
Nothing looks out of place here to me, sparing some comment spacing, thanks for putting in the PV doc descriptors 🎉

**Pull request merged**